xrpl/models/transactions/__init__.py

```diff
@@ -111,6 +113,7 @@
     "CheckCash",
     "CheckCreate",
     "Clawback",
+    "OracleDelete",
```

Comment:
This should be alphabetized

Reply:
fixed in [df462ba](https://github.com/XRPLF/xrpl-py/pull/701/commits/df462ba5ca42b12e4ab754c9c6446b990264f6da)

Reply:
fixed in [df462ba](https://github.com/XRPLF/xrpl-py/pull/701/commits/df462ba5ca42b12e4ab754c9c6446b990264f6da)

Reply:
I started out with the rippled names. But when I saw the conflict with defs.json, I only changed a few places 😅 , forgot the others. thanks